app/assets/stylesheets/components/buttons.scss

```diff
@@ -76,6 +77,12 @@
   }
 }
 
+// Icon alone
+.crayons-btn--icon-alone {
```

Comment:
`-alone`, so sad 😢

Reply:
hey, i could call it `.crayons-btn--icon-forever-alone` :D